### ISSUE TITLE
GU0014 should warn when using member in arithmetic expressions.

### ISSUE THREAD

**Issue by JohanLarsson** [GuOrg/Gu.Analyzers#165]
```cs
namespace RoslynSandbox
{
    class Foo
    {
        private readonly int n;

        public Foo(int n)
        {
            this.n = n;
            var square = this.n * this.n;
        }
    }
}
```
